uo_sys_framework/ACREModule/cfgAttributeCategories/Object.hpp

```diff
@@ -94,5 +94,77 @@ class UO_FW_ACREOptions {
 			condition = "objectBrain";
 			defaultValue = "0";
 		};
+		class UO_FW_ACRE_RACK_ENABLED {
+			property = "UO_FW_ACRE_RACK_ENABLED";
+			displayName = "Add Rack to Object";
+			tooltip = "Allows objects to be given radio racks defined by ACRE settings";
+			control = "CheckBox";
+			expression = "_this setVariable ['%s',_value,true]; if (_value) then {[{CBA_missionTime > 1},{[_this] call UO_FW_FNC_addRack;},[_this],30,{(format [""(ACRE_SETUP_MODULE) [ERR]: CBA_missionTime COULD NOT BE FOUND AFTER 30 SECONDS, UO_FW_FNC_addRack TIMED OUT""]) call UO_FW_FNC_DebugMessage;}] call CBA_fnc_WaitUntilAndExecute;};";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "false";
+		};
+		class UO_FW_ACRE_RACK_TYPE {
+			property = "UO_FW_ACRE_RACK_TYPE";
+			displayName = "Type of Rack";
+			tooltip = "Determines what kind of rack to add.";
+			control = "UO_FW_ACRE_Rack_Type";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "'""ACRE_VRC103""'";
+		};
+		class UO_FW_ACRE_NAMES {
+			property = "UO_FW_ACRE_NAMES";
+			displayName = "Add Rack Names";
+			tooltip = "Gives racks a custom Long and short name. Short name must be four or less characters.";
+			control = "EditShort";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "'[""Custom Rack Name"",""Rack""]'";
+		};
+		class UO_FW_ACRE_MOUNTED_RADIO {
+			property = "UO_FW_ACRE_MOUNTED_RADIO";
+			displayName = "Add Mounted Radio";
+			tooltip = "Add a radio to the rack by default.";
+			control = "CheckBox";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "false";
+		};
+		class UO_FW_ACRE_MOUNTED_RADIO_REMOVABLE {
+			property = "UO_FW_ACRE_MOUNTED_RADIO_REMOVABLE";
+			displayName = "Mounted Radio Removable";
+			tooltip = "Determines whether radios added by default are removable.";
+			control = "CheckBox";
+			expression = "_this setVariable ['%s',_value,true];";
+			condition = "objectVehicle + (object * (1-objectBrain)) + (1 - objectControllable)";
+			defaultValue = "false";
+		};
+		class UO_FW_ACRE_WHITELISTED_POSITIONS {
```

Comment:
Once @PiZZAD0X has his new Controls for array's merged, I will be integrating them into this attribute.